addons/wardrobe/functions/fnc_replaceContainer.sqf

```diff
@@ -0,0 +1,57 @@
+#include "../script_component.hpp"
+/*
+ * Author: OverlordZorn
+ * Function to replace a units wearable container while maintaining the content of the container.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Desired variant as classname <STRING>
+ * 2: Type of wearable container <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, "U_B_CTRG_3", UNIFORM] call ace_wardrobe_fnc_replaceContainer
+ * [player, "U_B_CTRG_1", UNIFORM] call ace_wardrobe_fnc_replaceContainer
+ *
+ * Public: No
+ */
+
+params ["_player", "_classTarget", "_equipmentType"];
+
+toFixed 0;
+private _magazineDetails = (magazinesAmmoFull _player) apply { [_x#0, _x#-2, _x#-1] };
```

Comment:
If new parameters are appended to the end of the command's return, this will break, so use positive indices.